lib/routes/password.js

```diff
@@ -155,6 +155,18 @@ module.exports = function (
           )
           .then(
             function () {
+              return db.account(passwordChangeToken.uid)
```

Comment:
Ditto here, but it doesn't look like `passwordChangeToken` currently sports an `email` field, so it would require a database migration.  Maybe not worth it for initial release, since this is a pretty uncommon operation.